tests/src/tests/mod.rs

```diff
@@ -1,13 +1,22 @@
 use crate::sandbox::{aurora::Aurora, fungible_token::FungibleToken, Sandbox};
 use aurora_engine_types::types::Address;
 use aurora_forwarder_factory::DeployParameters;
+use lazy_static::lazy_static;
```

Comment:
Could use [once_cell](https://docs.rs/once_cell/1.19.0/once_cell/sync/struct.Lazy.html) instead of lazy_static. They're basically the same API, but `once_cell` does not use any macros and there is [a version of it](https://doc.rust-lang.org/std/sync/struct.LazyLock.html) that will be in the standard library eventually.